apps/music/test/unit/player_view_test.js

```diff
@@ -0,0 +1,105 @@
+/* global PlayerView */
+
+'use strict';
+
+require('/js/ui/views/player_view.js');
+require('/shared/js/l10n.js');
```

Comment:
An example of that is here for example: https://github.com/mozilla-b2g/gaia/blob/master/apps/system/test/unit/app_chrome_test.js